### PR TITLE
Updated search-e2e prow

### PR DESCRIPTION
**Pull request by dislbenn**
Signed-off-by: dislbenn <lavontae.bennett@gmail.com>

### CODE REVIEW

File: ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml [hunk 1]
```diff
@@ -72,6 +72,16 @@ tests:
...
       REGISTRY_ORG: stolostron
     workflow: ocm-ci-image-mirror
+- as: latest-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: search-e2e-test
+    env:
+      IMAGE_REPO: search-e2e-test
+      IMAGE_TAG: latest
```
Comment (SherinV) [inline ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml:82]: Do we use the latest tag for testing?

**Review by SherinV — APPROVED** [review]

File: ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml (hunk 1 above)
Comment (dislbenn) [reply ci-operator/config/stolostron/search-e2e-test/stolostron-search-e2e-test-main.yaml:82]: The latest tag is being used for the image that is published to quay.

**Pull request merged**